index.js

```diff
@@ -44,17 +44,28 @@ var argv = yargs.scriptName('cypress-utils')
     global: true,
   })
   .option('configFile', {
-    alias: ['c', 'config'],
+    alias: ['C', 'config-file'],
     type: 'string',
     description: 'Path to the config file to be used.\nIf false is passed, no config file will be used.',
-    default: 'cypress.json',
+    default: 'cypress.config.js',
+    global: true,
+  })
+  .option('config', {
+    alias: ['c', 'config'],
+    type: 'string',
+    description: 'Set configuration values. Separate multiple values with commas. The values set here override any values set in your configuration file.',
     global: true,
   })
-  .option('integrationFolder', {
+  .option('specPattern', {
     alias: ['i', 'integration'],
     type: 'string',
-    description: 'Path to folder containing integration test files',
-    default: 'cypress/integration',
+    description: 'Path to folder containing e2e test files',
```

Comment:
```suggestion
    description: 'A glob pattern of the test files to load.',
```

Reply:
Pulled this description from https://docs.cypress.io/guides/references/configuration#e2e but reworded it to remove support for Array type.

Reply:
Thanks for the suggestion Trent Rand. I'll do the change. It's better.